src/System.Net.WebSockets/tests/WebSocketTests.cs

```diff
@@ -129,7 +129,11 @@ public static void IsStateTerminal_TerminalReturnsTrue(WebSocketState state)
         [InlineData(WebSocketState.Open, new WebSocketState[] { WebSocketState.Aborted, WebSocketState.CloseSent })]
         public static void ThrowOnInvalidState_ThrowsIfNotInValidList(WebSocketState state, WebSocketState[] validStates)
         {
-            Assert.Throws<WebSocketException>(() => ExposeProtectedWebSocket.ThrowOnInvalidState(state, validStates));
+            WebSocketException wse = Assert.Throws<WebSocketException>(() => ExposeProtectedWebSocket.ThrowOnInvalidState(state, validStates));
+            if (!PlatformDetection.IsFullFramework) // https://github.com/dotnet/corefx/issues/30779
```

Comment:
Might need to skip this test on UAP as well.

Reply:
Ok.  I'll just split it out into its own test and annotate it with skips for these.  And since I'm doing that, I'll address the other call sites, too.

Reply:
Ok. Thanks.  Looks good.  You should run Outerloop before merging.